### DISCUSSION TITLE
Error during authentication

### DISCUSSION THREAD

**Discussion by flower87lucky** [outline/outline#5537] [Self Hosting]
I used iso-server. 
outline logs:
{"level":"error","message":"Error during authentication"}
{"label":"http","level":"info","message":"  \u001b[90m-->\u001b[39m \u001b[1mGET\u001b[22m \u001b[90m/auth/oidc.callback?code=50a3WmWWQrXFyoae&state=d2baa18155f1f369\u001b[39m \u001b[36m302\u001b[39m \u001b[90m12ms\u001b[39m \u001b[90m69b\u001b[39m"}
{"label":"http","level":"info","message":"  \u001b[90m<--\u001b[39m \u001b[1mGET\u001b[22m \u001b[90m/?notice=auth-error\u001b[39m"}
{"label":"database","level":"debug","message":"Executing (default): SELECT \"id\", \"name\", \"subdomain\", \"domain\", \"defaultCollectionId\", \"avatarUrl\", \"sharing\", \"inviteRequired\", \"signupQueryParams\", \"guestSignin\", \"documentEmbeds\", \"memberCollectionCreate\", \"defaultUserRole\", \"preferences\", \"createdAt\", \"updatedAt\", \"deletedAt\" FROM \"teams\" AS \"team\" WHERE (\"team\".\"deletedAt\" IS NULL) LIMIT 1;"}
{"label":"http","level":"info","message":"  \u001b[90m-->\u001b[39m \u001b[1mGET\u001b[22m \u001b[90m/?notice=auth-error\u001b[39m \u001b[32m200\u001b[39m \u001b[90m13ms\u001b[39m \u001b[90m1.55kb\u001b[39m"}
{"label":"http","level":"info","message":"  \u001b[90m<--\u001b[39m \u001b[1mPOST\u001b[22m \u001b[90m/api/auth.config\u001b[39m"}
{"label":"database","level":"debug","message":"Executing (default): SELECT \"team\".*, \"authenticationProviders\".\"id\" AS \"authenticationProviders.id\", \"authenticationProviders\".\"name\" AS \"authenticationProviders.name\", \"authenticationProviders\".\"enabled\" AS \"authenticationProviders.enabled\", \"authenticationProviders\".\"providerId\" AS \"authenticationProviders.providerId\", \"authenticationProviders\".\"teamId\" AS \"authenticationProviders.teamId\", \"authenticationProviders\".\"createdAt\" AS \"authenticationProviders.createdAt\" FROM (SELECT \"team\".\"id\", \"team\".\"name\", \"team\".\"subdomain\", \"team\".\"domain\", \"team\".\"defaultCollectionId\", \"team\".\"avatarUrl\", \"team\".\"sharing\", \"team\".\"inviteRequired\", \"team\".\"signupQueryParams\", \"team\".\"guestSignin\", \"team\".\"documentEmbeds\", \"team\".\"memberCollectionCreate\", \"team\".\"defaultUserRole\", \"team\".\"preferences\", \"team\".\"createdAt\", \"team\".\"updatedAt\", \"team\".\"deletedAt\" FROM \"teams\" AS \"team\" WHERE (\"team\".\"deletedAt\" IS NULL) LIMIT 1) AS \"team\" LEFT OUTER JOIN \"authentication_providers\" AS \"authenticationProviders\" ON \"team\".\"id\" = \"authenticationProviders\".\"teamId\";"}
{"label":"http","level":"info","message":"  \u001b[90m-->\u001b[39m \u001b[1mPOST\u001b[22m \u001b[90m/api/auth.config\u001b[39m \u001b[32m200\u001b[39m \u001b[90m11ms\u001b[39m \u001b[90m108b\u001b[39m"}

iso-server logs:
172.17.0.1 - - [07/Jul/2023:07:03:03 +0000] "GET /dialog/authorize?response_type=code&redirect_uri=http%3A%2F%2F10.0.0.253%3A13090%2Fauth%2Foidc.callback&scope=openid%20profile%20email&state=dc28d1f69fcb8aeb&client_id=b8c40013-cc03-4bc5-b3a5-6a31046fa415 HTTP/1.1" 200 7665 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
172.17.0.1 - - [07/Jul/2023:07:03:04 +0000] "POST /dialog/authorize/decision HTTP/1.1" 302 226 "http://10.0.0.253:3000/dialog/authorize?response_type=code&redirect_uri=http%3A%2F%2F10.0.0.253%3A13090%2Fauth%2Foidc.callback&scope=openid%20profile%20email&state=dc28d1f69fcb8aeb&client_id=b8c40013-cc03-4bc5-b3a5-6a31046fa415" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"
172.17.0.1 - - [07/Jul/2023:07:03:04 +0000] "POST /oauth/token HTTP/1.1" 401 - "-" "Node-oauth"

**Comment by tommoor** [comment]
You are not running in production mode, `NODE_ENV=production` to get better debugging here.
